core/cloudflow-localrunner/src/main/scala/cloudflow/localrunner/LocalRunner.scala

```diff
@@ -80,13 +82,16 @@ object LocalRunner extends StreamletLoader {
     JRuntime.getRuntime.addShutdownHook(shutdownHook(fos))
 
     Console.withOut(fos) {
-      System.setOut(new PrintStream(fos))
-      readDescriptorFile(appDescriptorFilename) match {
-        case Success(applicationDescriptor) ⇒
-          run(applicationDescriptor, localConfig)
-        case Failure(ex) ⇒
-          log.error(s"Failed JSON unmarshalling of application descriptor file [${appDescriptorFilename}].", ex)
-          System.exit(1)
+      Console.withErr(fos) {
+        System.setOut(new PrintStream(fos))
```

Comment:
Is it possible to get the Streams from DI or as parameters? This will make(possibly) the behavior testable with a trick similar to: https://github.com/lightbend/cloudflow/issues/771#issue-717214138